app/src/main/kotlin/com/wire/android/notification/CallNotificationManager.kt

```diff
@@ -70,15 +72,42 @@ class CallNotificationManager @Inject constructor(
                     }
                 }
         }
+        scope.launch {
+            outgoingCallForUsers.collectLatest {
+                if (it.isEmpty()) {
+                    hideOutgoingCallNotification()
+                } else {
+                    it[it.keys.first()]?.let { call ->
+                        showOutgoingCallNotification(
+                            conversationId = call.conversationId,
+                            userId = it.keys.first(),
+                            conversationName = call.conversationName
+                                ?: context.getString(R.string.calling_participant_tile_default_user_name)
+                        )
+                    }
+                }
```

Comment:
Maybe a tiny bit more readable and less indented.
```suggestion
                if (it.isEmpty()) {
                    hideOutgoingCallNotification()
                } else {
                    val call = it.values.first()
                    showOutgoingCallNotification(
                        conversationId = call.conversationId,
                        userId = it.keys.first(),
                        conversationName = call.conversationName
                            ?: context.getString(R.string.calling_participant_tile_default_user_name)
                    )
                }
```